votey/utils.py

```diff
@@ -14,4 +14,14 @@
 def batch(lst: List[T], n: int = 1) -> Iterable[List[T]]:
     ln = len(lst)
     for ndx in range(0, ln, n):
-        yield lst[ndx:min(ndx + n, ln)]
\ No newline at end of file
+        yield lst[ndx:min(ndx + n, ln)]
+
+def get_footer(user_id: str, anonymous: bool, secret: bool, explode: bool) -> str:
+    if explode:
+        return f'This poll and its data will be scrubbed in 30 minutes (not yet implemented)'
+    if secret:
+        return f'Author is hidden'
```

Comment:
Can these not be combined? As-is, it'll return only the first one matched, but a poll could be both secret and explode (for example), right?

Reply:
That's correct. In my mind I built this with the idea that anonymous is a subset of secret which is a subset of explode. But yeah these strings should maybe be concat'd for those cases